### PR TITLE
Feature/issue 1402 1405 1088 1393 multitenancy

### PR DESCRIPTION
**Pull request by erlemantos**
First pass of multitenancy

### CODE REVIEW

**Comment by aedelmann** [comment]
Hi @erlemantos 
Does this PR contain any 3rd party code or libraries that need CQ approval ?
If yes, please list them in this PR. 
Thanks.

**Comment by erlemantos** [comment]
> Hi @erlemantos
> Does this PR contain any 3rd party code or libraries that need CQ approval ?
> If yes, please list them in this PR.
> Thanks.

No. This PR doesn't contain any third party code.

**Comment by aedelmann** [comment]
Hi @erlemantos Can u resolve the conflicts? thanks

File: repository/repository-core/src/main/java/org/eclipse/vorto/repository/tenant/ITenantService.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ */
+package org.eclipse.vorto.repository.tenant;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+import org.eclipse.vorto.repository.core.IUserContext;
+import org.eclipse.vorto.repository.domain.Tenant;
+
+// TODO : Document interface
```
Comment (aedelmann) [inline repository/repository-core/src/main/java/org/eclipse/vorto/repository/tenant/ITenantService.java]: @erlemantos Can you resolve the ToDo

Comment (erlemantos) [reply repository/repository-core/src/main/java/org/eclipse/vorto/repository/tenant/ITenantService.java]: Done.

**Pull request merged**